### ISSUE TITLE
Add email recipient suggestions

### ISSUE THREAD

**Issue by marzooqy** [thunderbird/thunderbird-android#8328]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.904

### Problem you are trying to solve

It takes a while to figure out and type the email addresses of the individuals that I intend to send an email to.

### Suggested solution

Outlook adds all of the emails of anyone that you've interacted with as autocomplete suggestions in a dropdown menu. I think that would nice to have in thunderbird/k9mail.

**Comment by rezazarchi** [comment]
Thunderbird/K9mail use your device's contacts list to suggest recipients. To get the contacts you need, sync them using an app that works with your mail server type (like DAVx⁵, EteSync, Exchange, or others).

**Comment by kewisch** [comment]
Hi @marzooqy , would @rezazarchi's suggestion work for you? It does indeed sound like this would cover the autocomplete you are looking for.

**Comment by marzooqy** [comment]
Yes, I think it would be enough to just add the common emails to my contacts. Although automatic suggestions would be more convenient.

**Comment by kewisch** [comment]
Desktop has this "collected addresses" address book that automatically adds some addresses to the contacts, I think it would be helpful for this use case. Would you mind creating an idea around this on https://connect.mozilla.org/t5/ideas/idb-p/ideas/label-name/thunderbird%20android ? I think this would make a great idea that people can vote on.

**Comment by marzooqy** [comment]
> Desktop has this "collected addresses" address book that automatically adds some addresses to the contacts, I think it would be helpful for this use case. Would you mind creating an idea around this on https://connect.mozilla.org/t5/ideas/idb-p/ideas/label-name/thunderbird%20android ? I think this would make a great idea that people can vote on.

[Done.](https://connect.mozilla.org/t5/ideas/add-auto-complete-email-recipient-suggestions-for-thunderbird-on/idi-p/74937)

**Comment by Artim96** [comment]
I'd second this request. Adding all mail addresses you interact with is a workaround, but a bad one at best. You'd bloat your contacts list with people you may have frequent interaction with, but maybe only for a short period. So you'd have to manually go through your contacts to clean them up. Having a dedicated contacts store in Thunderbird - like on desktop - would allow to automatically suggest any address you have corresponded with. Of course, ideally that would exclude all spam mail and would discard them after a certain period of time.
